src/redturtle/prenotazioni/restapi/services/booking/update.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+import json
+
+from plone import api
+from plone.restapi.services.content.update import ContentPatch
+from zExceptions import BadRequest
+from zope.component import getUtility
+from zope.schema._bootstrapinterfaces import ValidationError
+from zope.schema.interfaces import IVocabularyFactory
+
+from redturtle.prenotazioni import _
+
+
+class UpdateBooking(ContentPatch):
+    def reply(self, *args, **kwargs):
+
+        self.handle_additional_fields()
+
+        return super().reply(*args, **kwargs)
+
+    def handle_additional_fields(self):
+        data = json.loads(self.request.get("BODY", b"").decode())
```

Comment:
there is an utility in plone.restapi to get form data

Reply:
done